c2corg_ui/static/partials/imageuploader.html

```diff
@@ -20,8 +20,9 @@
         <!-- EXIF INFO-->
         <div class="exif" ng-style="file.metadata.FocalLength && {'opacity': 1} || {'opacity' : 0}">
           <p ng-if="file.metadata.DateTime"><span class="glyphicon glyphicon-calendar"></span><span>{{file.metadata.DateTime}}</span></p>
-          <p><span class="glyphicon glyphicon-certificate"></span><span>{{file.metadata.FocalLength}}f</span>&nbsp;
-            <span ng-if="file.metadata.ShutterSpeedValue">1/{{file.metadata.ShutterSpeedValue}}s</span> &nbsp; 
+          <p><span class="glyphicon glyphicon-certificate"></span><span>f/{{file.metadata.FNumber}}</span>&nbsp;
+            <span ng-if="file.metadata.ExposureTime">1/{{1/file.metadata.ExposureTime}}s</span> &nbsp;
+            <span ng-if="file.metadata.FocalLengthIn35mmFilm">{{file.metadata.FocalLengthIn35mmFilm}}mm</span> &nbsp;
             <span><b>ISO:</b> <span>{{file.metadata.PhotographicSensitivity}}</span></span>
```

Comment:
Also: not sure this ISO data is available (for instance if the image is a drawing) => don't display the info line if the info is not available.